src/lib/middleware/morgan.ts

```diff
@@ -26,12 +26,22 @@ function skipAssets(req: ArtsyRequest, res: ArtsyResponse): boolean {
   )
 }
 
+const BANNED_PARAMS = ["setup_intent_client_secret"]
```

Comment:
#superminor - a line-break between the variable and function declaration would make this nicer to read